ethereum/core/src/testFixtures/java/tech/pegasys/teku/core/ChainBuilder.java

```diff
@@ -93,10 +93,15 @@ private ChainBuilder(
     existingBlocks.values().forEach(b -> blocksByHash.put(b.getRoot(), b));
   }
 
+  @Deprecated // Should supply a Spec value
   public static ChainBuilder createDefault() {
     return ChainBuilder.create(TestSpecFactory.createMinimalPhase0(), DEFAULT_VALIDATOR_KEYS);
   }
```

Comment:
We're deprecating a lot of stuff which is creating a lot of warnings.  I wonder if we'd be better off just having a `TestSpecFactory.createDefault()` that we use here and not always need to specify the spec explicitly.  Otherwise we have to update every test which is going to take us ages.

Reply:
My concern is that it's easy to miss the spec dependency here.  I see a lot of tests that are setting up a `Spec` object and then not passing it across to the various test utilities, so it would be easy to get a mismatch.

Looks like there are about 20 usages of this - I could probably just go through and fix them now ...

Reply:
Just went ahead and cut the deprecated method.

Reply:
Also added `TestSpecFactory.createDefault()` :)

Reply:
Nice. :)